src/rules/no-cycle.js

```diff
@@ -62,6 +68,8 @@ module.exports = {
       context,
     );
 
+    const scc = StronglyConnectedComponentsBuilder.get(myPath, context);
```

Comment:
Can this row and line 114-117 move after line 119?
when skipErrorMessagePath is false, there is no need to do scc check.

Reply:
the scc check is how we know if there's cycles or not.

Reply:
How do we move forward with this? Can we resolve this conversation?